### PR TITLE
Fix nomad dependency

### PR DESCRIPTION
**Pull request by ladinesa**

### CODE REVIEW

**Review by blueraft — APPROVED** [review]
Looks good to me. 

Perhaps worth mentioning this step in the README in the local development section.

**Comment by ladinesa** [comment]
> Looks good to me.
> 
> Perhaps worth mentioning this step in the README in the local development section.

will include in the nomad mr, waiting for approval from @JosePizarro3 for other parsers.

File: .github/workflows/python-actions.yaml [hunk 1]
```diff
@@ -13,16 +12,40 @@ jobs:
...
+        python-version: 3.9
+    - name: Build the package
+      run: |
+        pip install --upgrade pip
+        pip install build
+        python -m build --sdist
+    - name: Install the package
+      run: |
+        pip install dist/*.tar.gz --index-url https://gitlab.mpcdf.mpg.de/api/v4/projects/2187/packages/pypi/simple
   ruff-linting:
```
Comment (JosePizarro3) [inline .github/workflows/python-actions.yaml:49]: would you mind changing the `ruff-formatting` for proper Ruff format check (this Ahmed and I talked about this already)?
```
          args: "format . --check"
          version: 0.1.8
```

**Review by JosePizarro3 — APPROVED** [review]
Looks good to me. Did you check the setup script in NOMAD with these changes? 🙂 

Thanks a lot!

File: .github/workflows/python-actions.yaml (hunk 1 above)
Comment (ladinesa) [reply .github/workflows/python-actions.yaml:49]: I did not add this ruff format, but I would remove autoformat from the actions, personally, I would put the burden of formatting on the developer.

Comment (JosePizarro3) [reply .github/workflows/python-actions.yaml:49]: Yeah, I had the same in mind, but @blueraft made the point that the format could be checked anyways in the pipeline to ensure consistency.

In any case, you can leave it also from this prs, like you prefer.

**Pull request merged**